setup.py

```diff
@@ -39,6 +39,7 @@
         'six>=1.7.0',
         'tqdm',
         'toml',
+        'dateparser'
```

Comment:
This could be avoided by using a specific format and parsing that, but ofc, this offers more flexibility. Should I remove this?

Reply:
dateparser is a nice and very convenient library, I don't mind to add it, we could use it in the other commands later if needed :+1: